external/grpc/Kconfig.protocol

```diff
@@ -10,3 +10,21 @@ config GRPC
 	depends on HAVE_CXX
 	---help---
 		Enable support for the gRPC.
+
+if GRPC
+
+comment "gRPC Config Parameters"
```

Comment:
"Config Parameters" is unnecessary comment. All of config are config parameters.
How about "gRPC Security Parameters"?

Reply:
And to show those configs, how about making GRPC as menuconfig?

Reply:
@sunghan-chang I removed the commend as you said. I will add new comment if there would be more than 2 options in the future,